src/Controller/Admin/ActivityController.php

```diff
@@ -104,13 +104,19 @@ public function groupAction(Group $group): Response
     public function newAction(Request $request, GroupRepository $groupRepo): Response
     {
         $activity = new Activity();
+        $price = new PriceOption();
+        $activity->AddOption($price);
 
         $form = $this->createForm('App\Form\Activity\ActivityNewType', $activity);
         $form->handleRequest($request);
 
         if ($form->isSubmitted() && $form->isValid()) {
+            $price
```

Comment:
The form is responsible for updating any value, but if (in the future) details or the confirmation message are added to the form, this code will break. Therefore, please move these 3 lines to ln. 109, as this will improve the resiliency of the code